inference-engine/src/readers/ir_reader/CMakeLists.txt

```diff
@@ -29,6 +29,8 @@ target_compile_definitions(${TARGET_NAME} PRIVATE IMPLEMENT_INFERENCE_ENGINE_PLU
 target_include_directories(${TARGET_NAME} PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}")
 
 target_link_libraries(${TARGET_NAME} PRIVATE ngraph
+                                             ir_frontend
```

Comment:
I propose to remove reader and use frontend instead.

Reply:
We can't remove reader API just because we have +100 tests that use old IR version. But @mbencer already updated ReadNetwork to use FrontendManager. So we decided to merge this PR first and then IR FE will be enabled in Mateusz PR.

Reply:
Can we remove IR reader for IR v10 at least?